src/main/java/org/opentripplanner/middleware/utils/CommandLineProcessor.java

```diff
@@ -0,0 +1,120 @@
+package org.opentripplanner.middleware.utils;
+
+import org.opentripplanner.middleware.recurringjobs.RecurringJob;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+import static org.opentripplanner.middleware.utils.ConfigUtils.DEFAULT_ENV;
+
+/**
+ * Parses the command line to extract which jobs and/or services should be started. The only stipulation, is that if a
+ * config file is defined, it must be the first argument defined. e.g.
+ *
+ * /configurations/test/env.yml --endpoints-only yes --recurring-jobs monitor-all-trips-job
+ */
+public class CommandLineProcessor {
+    public static final List<String> END_POINTS_ONLY_FLAGS = List.of("--endpoints-only", "-E");
```

Comment:
Sorry, rename to `--endpoints` because the flag is not meant to exclude the other features.